engine/src/main/java/com/datatorrent/stram/webapp/StramWebServices.java

```diff
@@ -511,6 +532,16 @@ public JSONObject getLogicalOperators() throws Exception
     nodeList.operators = dagManager.getLogicalOperatorInfoList();
     return new JSONObject(objectMapper.writeValueAsString(nodeList));
   }
+  
+  @GET
+  @Path(PATH_LOGICAL_PLAN_MODULES)
+  @Produces(MediaType.APPLICATION_JSON)
+  public JSONObject getLogicalModules(@QueryParam("flatten") String flatten) throws Exception
```

Comment:
Yes, Initially I made them as boolean and Boolean but it was not working, the value of flatten variable was not getting set. Hence changed it to String. 
Do you know how to make boolean work with the java ws.